src/components/Mission.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prop-types
+const Mission = ({ name, description }) => (
+  <div className="mission-container ">
+    <div className="column-mission">{name}</div>
+    <div className="column-description">{description}</div>
+    <div className="column-status"><span>Member</span></div>
+    <div className="column-status-2"><span>Mission</span></div>
+  </div>
```

Comment:
React-Bootstrap has a nice way of setting up these as __rows__ in a `Grid system`.
I'll leave a [link](https://react-bootstrap.github.io/layout/grid/) in case you're interested in trying it out. 😉